src/modules/dapp/hooks/useTransactionSocket.ts

```diff
@@ -73,12 +73,11 @@ export const useTransactionSocket = (isWorkspaceReady: boolean) => {
       setTx(tx);
       setValidAt(validAt);
       summary.getTransactionSummary({
-        providerUrl: vault.provider,
         transactionLike: tx,
         from: vault.address,
       });
     });
-  }, [socket.connected, isWorkspaceReady]);
```

Comment:
Those dependencies was removed to avoid unnecessary retries/requests. 
With those, we had a lot of requests and some times, we got a http error `429 too many requests`.